connectivity/check/action.go

```diff
@@ -519,6 +521,12 @@ retry:
 			err = fmt.Errorf("no flows returned")
 		}
 		if err := w.Retry(err); err != nil {
+			if len(flows) == 0 {
+				retriesOnEmpty--
```

Comment:
maybe log an info here? might be interesting to know when we are hitting this case.

Reply:
Added the log line :-)